content/blogposts/2021/how-to-search-code-with-sourcegraph-a-cheat-sheet.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "How to search code with Sourcegraph — a cheat sheet"
+tags: [blog]
+publishDate: 2021-08-19T18:00+02:00
+description: "Sourcegraph’s new admin analytics make it easier for admins to understand user engagement, measure efficiencies, and quantify the value of Sourcegraph."
```

Comment:
Should we copy over the original post's description?

```suggestion
description: "A guide to help you get up to speed with Sourcegraph's commands quickly"
```

Reply:
Thanks, good catch - I was looking at another blog to make sure I got the right meta fields.